pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -256,6 +256,7 @@ let
           sed -i 's@include/pybind11@pybind11@g' $src
         done
       '' + lib.optionalString cudaSupport ''
+        export NIX_LDFLAGS+=" -L${backendStdenv.nixpkgsCompatibleLibstdcxx}/lib"
```

Comment:
Does the order matter here? I'm assuming `stdenv`'s (undesirable) libstdc++ will also be in `NIX_LDFLAGS` somewhere?

Reply:
I don't think there's any extra `-L` in `NIX_LDFLAGS`, but maybe I should quickly check it by adding an `echo` somewhere